src/game/mod.rs

```diff
@@ -44,6 +45,7 @@ impl Game {
     pub fn load() -> Result<Self, Error> {
         let data = fs::read(data_file()).or(Err(Error::NoDataFile))?;
         let game: Game = bincode::deserialize(&data).unwrap();
+        let _config = character::CharacterConfig::load(config_file());
```

Comment:
consider doing this directly in main, so tha game module doesn't need to know about it